src/controllers/userProfileSettings.js

```diff
@@ -0,0 +1,47 @@
+import bcrypt from 'bcrypt';
+// import Jwt from 'jsonwebtoken';
+import User from '../models/user.model.js';
+import redisClient from '../helpers/redis.js';
+
+const { hash } = bcrypt;
+
+const UpdateUserInfo = async (req, res) => {
+  const userId = req.user.id;
+  const {
+    username,
+    // email,
+    password,
```

Comment:
A user should not be able to update password like this. It needs the email flow and for our cases when they get notified that the password needs to be changed or when they need to reset

Reply:
yes, let me do it